source/versions/filenames.rst

```diff
@@ -57,22 +63,6 @@ version ``1.12.2``, was built with Forge ``14.23.4.2705`` (Build ``2705``), guar
 ``7.1.0``, is not a recommended version, and is build number ``3449`` in development, which will be the ``5`` th 
 release of SpongeForge once this version is released. 
 
-.. note::
-
-    Normal Forge mods can usually run on any build of Forge for a given Minecraft version (e.g. 1.8.0) without any
-    problem. However, SpongeForge needs to access, among other things, internal parts of Forge, which most mods
-    shouldn’t be touching, let alone modifying as Sponge does. Since Forge is free to change their internal code
-    whenever they want to, its normal guarantee of backwards-compatibility doesn’t apply to SpongeForge. Feel free to
-    use more recent versions of Forge, than the one used for SpongeForge, but we can't always guarantee compatibility.
-
-.. warning::
-
```

Comment:
this still applies, why did you remove it?

Reply:
Oh, snap! At first, I saw it as a duplication with the new content. Had I seen the issue tracker part, I would have misunderstood a previous comment to mean not to use the issue tracker. I will put it back in.

Reply:
Corrected.